topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md

```diff
@@ -0,0 +1,160 @@
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Access the Galaxy server
+
+> <hands-on-title>Log in or register</hands-on-title>
+>
+> 1. Open your favorite browser (Chrome/Chromium, Safari, or Firefox, but not Internet Explorer/Edge!)
+> 2. Browse to [Galaxy](https://usegalaxy.eu) 
+> 3. Choose *Login or Register* from the navigation bar at the top of the page
+> 4. If you have previously registered an account with this particular instance of Galaxy (user accounts are *not* shared between public servers!), proceed by logging in with your registered *public name*, or email address, and your password.
+>
+>    If you need to create a new account, click on *Register here* instead.
+>
+
+# Upload data on Galaxy
+
+> <hands-on-title> Data Upload </hands-on-title>
+>
+> 1. Create a new history for this tutorial and give it a proper name
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>    {% snippet faqs/galaxy/histories_rename.md %}
+
+
+# Perform NCBI Blast+ on Galaxy 
+
+> After successfully logging in to the Galaxy server, Go to the [NCBI-Blast+](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/ncbi_blast_plus/ncbi_blastp_wrapper/2.10.1+galaxy2) tool.  
+> Since MAdLandDB is the collection of protein sequences, You can perform [BLASTp](toolshed.g2.bx.psu.edu/repos/devteam/ncbi_blast_plus/ncbi_blastp_wrapper/2.10.1+galaxy2) and [BLASTx](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/ncbi_blast_plus/ncbi_blastx_wrapper/2.10.1+galaxy2) tools.
+
+>
+> <hands-on-title> Similarity search against MAdLand Database </hands-on-title>
+>
+> 1. {% icon tool %} As input file, select the protein sequences 
+> 2. Choose the protein BLAST database *MAdLandDB* with the following parameters:
+>    - In *"input Options"*:
+>        - {% icon param-file %} *"Protein query sequence(s)"*: `output` (Input dataset)
+>        - *"Subject database/sequences"*: `Locally installed BLAST database`
+>        - *"Protein BLAST database"*: `MAdLandDB`
+>    - In *"Output Options"*:
+>       - *"Output format"*: `Tabular (extended 25 columns)` (optional)
+
+> <img src="../../images/ncbi-blast-against-the-madland/blast-example.png" alt="blast against madland" width="80%">
+
+# Blast output 
+
+>{% icon tool %} The following 12 columns are predicted in the tabular formatted output file of the Blast result (you can select desired output format options):  
```

Comment:
Once you have found a scientific question that can be used as starting point, I think we could try to explain the results in detail, in order to solve the initial problem.